tests/test_registry_management.py

```diff
@@ -330,6 +331,40 @@ def test_add_supplemental_dimension(tmp_registry_db):
     assert found_new_dimension
 
 
+def test_remove_dataset(tmp_registry_db):
+    test_project_dir, tmp_path, db_name = tmp_registry_db
+    mgr = make_test_data_registry(
+        tmp_path, test_project_dir, dataset_path=TEST_DATASET_DIRECTORY, database_name=db_name
+    )
+    project_mgr = mgr.project_manager
+    project_id = project_mgr.list_ids()[0]
+    config = project_mgr.get_by_id(project_id)
+    dataset_id = config.model.datasets[0].dataset_id
+    assert config.model.datasets[0].status == DatasetRegistryStatus.REGISTERED
+
+    runner = CliRunner(mix_stderr=False)
+    result = runner.invoke(
+        cli_admin,
+        [
+            "--username",
+            "root",
+            "--password",
+            DEFAULT_DB_PASSWORD,
+            "--database-name",
+            db_name,
+            "--offline",
+            "registry",
+            "datasets",
+            "remove",
+            dataset_id,
+            "1.0.0",
+        ],
+    )
+    assert result.exit_code == 0
+    config = project_mgr.get_by_id(project_id)
+    assert config.model.datasets[0].status == DatasetRegistryStatus.UNREGISTERED
```

Comment:
Is there anything else to check? It should still work to ask for the dataset_id since it's still in the project config, but are there any other data available from the project that should no longer be accessible and/or be None instead of actual data?

Reply:
Derived datasets would also be impacted. But I don't think this should ever be a normal activity outside of the case of "wipe this one out and start over."

Reply:
Do we know which derived datasets rely on which initial datasets? Can we remove those too?

Reply:
Check-in conversation: We do not have those tracking data available today.